<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="profileLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Profile Configuration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="opacityHorizontalLayout" stretch="0,10,0">
          <item>
           <widget class="QLabel" name="transparencyLabel">
            <property name="text">
             <string>Opacity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="transparencyHorizontalSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="transparencySpinBox">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="textRadioButton">
          <property name="text">
           <string>Use Text</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="imageRadioButton">
          <property name="text">
           <string>Use Image</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QStackedWidget" name="typeStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_5">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Text:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="fontLabel">
              <property name="text">
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QFontComboBox" name="fontComboBox">
              <property name="currentFont">
               <font>
                <family>Helvetica</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="sizeTextLabel">
              <property name="text">
               <string>Font Size:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sizeSpinBox">
              <property name="minimum">
               <number>6</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>32</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Font attributes:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="boldToolButton">
                <property name="toolTip">
                 <string>Bold</string>
                </property>
                <property name="text">
                 <string>B</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="italicToolButton">
                <property name="toolTip">
                 <string>Italic</string>
                </property>
                <property name="text">
                 <string>I</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="underlineToolButton">
                <property name="toolTip">
                 <string>Underline</string>
                </property>
                <property name="text">
                 <string>U</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Font Colors:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="textColorButton">
                <property name="text">
                 <string>Text Color...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="outlineColorButton">
                <property name="text">
                 <string>Outline Color...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="1">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_6">
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QLabel" name="selectImageForLabel">
              <property name="text">
               <string>Select image used for watermarking</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="watermarkLineEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="watermarkPushButton">
              <property name="text">
               <string>Select...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>267</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
